src/components/SingleProduct.js

```diff
@@ -0,0 +1,20 @@
+import React from "react";
+
+export function SingleProduct( {product} ) {
+  let stocked = 'No';
```

Comment:
This makes sense but you could just do
`let stocked = product.inStock ? 'Yes' : 'No';`

Reply:
Nice way to condense it  a bit. Haven't been using that notation much yet, but I'll keep it in mind.